src/ServerDriver_OSVR.cpp

```diff
@@ -86,9 +87,13 @@ vr::EVRInitError ServerDriver_OSVR::Init(vr::IVRDriverContext* driver_context)
 
     trackedDevices_.emplace_back(std::make_unique<OSVRTrackedHMD>(*(context_.get())));
     trackedDevices_.emplace_back(std::make_unique<OSVRTrackingReference>(*(context_.get())));
+    trackedDevices_.emplace_back(std::make_unique<OSVRTrackedController>(*(context_.get()), 0)); // left hand
```

Comment:
Instead of passing in `0` and `1` for the controller index, could we pass in `vr::TrackedControllerRole_LeftHand` and `vr::TrackedControllerRole_RightHand` instead? (They eval to 1 and 2 instead, so other code would have to be tweaked.)

I haven't reviewed the usage of the controller index yet, so this may be a bad idea.